webapp/views/App/views/Data/DataImport/DataImportCsvView.js

```diff
@@ -67,8 +87,23 @@ export const DataImportCsvView = () => {
         </FormItem>
       )}
 
+      <FormItem label={i18n.t('dataImportView.importType.label')}>
+        <ButtonGroup
+          selectedItemKey={dataImportType}
+          onChange={onDataImportTypeChange}
+          items={Object.values(importTypes).map((importType) => ({
+            key: importType,
+            label: i18n.t(`dataImportView.importType.${importType}`),
+          }))}
+        />
+      </FormItem>
+
       <FormItem label={i18n.t('dataImportView.importIntoEntity')}>
-        <EntitySelectorTree nodeDefUuidActive={selectedEntityDefUuid} onSelect={onEntitySelect} />
+        <EntitySelectorTree
+          nodeDefUuidActive={selectedEntityDefUuid}
+          onSelect={onEntitySelect}
+          isDisabled={() => dataImportType === importTypes.insertNewRecords}
```

Comment:
??

Reply:
isDisabled should be renamed into isItemDisabled or isNodeDefDisabled, because it's a function that takes a node def and tells if the corresponding item in the tree is disabled...

Reply:
but should be a function or just a variable?

Reply:
a function, then there should be a "disabled" property to disable the selection at all